File: src/test/java/ru/ezhov/rocket/action/infrastructure/ReflectionRocketActionUiRepositoryTest.java
package ru.ezhov.rocket.action.infrastructure;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class ReflectionRocketActionUiRepositoryTest {
    @Test
    public void test() {
        ReflectionRocketActionUiRepository repository = new ReflectionRocketActionUiRepository();

        assertEquals(12, repository.all().size());
    }
}

File: src/test/kotlin/ru/ezhov/rocket/action/infrastructure/ReflectionRocketActionUiRepositoryTest.kt
package ru.ezhov.rocket.action.infrastructure

import org.junit.Assert
import org.junit.Test

class ReflectionRocketActionUiRepositoryTest {
    @Test
    fun test() {
        val repository = ReflectionRocketActionUiRepository()
        Assert.assertEquals(19, repository.all().size.toLong())
    }
}